Processing/magicCircle.pde

```diff
@@ -33,7 +34,10 @@ class MagicCircle{
     magicSymbol.setPosition(centerPosition);
   }
   void update(){
+    pushMatrix();
```

Comment:
draw() could perhaps do the same thing as push and popping matrixes. Try that for a moment.